quick_deploy.sh

```
@@ -0,0 +1,65 @@
+# Set default values if not provided
+export LANGGRAPH_API_URL="${LANGGRAPH_API_URL:-https://ghl-langgraph-agent-95f8713b0bd6545391589bd2f0ca67eb.us.langgraph.app}"
+export LANGGRAPH_ASSISTANT_ID="${LANGGRAPH_ASSISTANT_ID:-agent}"
+
+echo "📋 Configuration:"
+echo "  API URL: $LANGGRAPH_API_URL"
+echo "  Assistant ID: $LANGGRAPH_ASSISTANT_ID"
+echo "  API Key: ***${LANGGRAPH_API_KEY: -4}"
+echo ""
+
+# Change to project directory
+cd langgraph-ghl-agent
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)